docs/reference/tab-widgets/inference-api/infer-api-task.asciidoc

```diff
@@ -93,4 +93,33 @@ NOTE: When using this model the recommended similarity measure to use in the
 embeddings are normalized to unit length in which case the `dot_product` and
 the `cosine` measures are equivalent.
 
-// end::openai[]
\ No newline at end of file
+// end::openai[]
+
+// tag::azure-openai[]
+
+[source,console]
+------------------------------------------------------------
+PUT _inference/text_embedding/azure-openai_embeddings <1>
```

Comment:
The `model_id`/`inference_id` is referenced as `azure_openai_embeddings` elsewhere.
```suggestion
PUT _inference/text_embedding/azure_openai_embeddings <1>
```